<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlConnectionDialog</class>
 <widget class="QDialog" name="SqlConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>D&amp;river:</string>
     </property>
     <property name="buddy">
      <cstring>driverCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="driverCombo">
     <property name="toolTip">
      <string>Select the database driver</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel3">
     <property name="text">
      <string>&amp;Database Name:</string>
     </property>
     <property name="buddy">
      <cstring>databasePicker</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="E5PathPicker" name="databasePicker" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Enter the database name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel4">
     <property name="text">
      <string>&amp;Username:</string>
     </property>
     <property name="buddy">
      <cstring>usernameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="usernameEdit">
     <property name="toolTip">
      <string>Enter the user name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textLabel4_2">
     <property name="text">
      <string>&amp;Password:</string>
     </property>
     <property name="buddy">
      <cstring>passwordEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="passwordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textLabel5">
     <property name="text">
      <string>&amp;Hostname:</string>
     </property>
     <property name="buddy">
      <cstring>hostnameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="hostnameEdit">
     <property name="toolTip">
      <string>Enter the hostname</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="textLabel5_2">
     <property name="text">
      <string>P&amp;ort:</string>
     </property>
     <property name="buddy">
      <cstring>portSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="portSpinBox">
     <property name="toolTip">
      <string>Enter the port number</string>
     </property>
     <property name="specialValueText">
      <string>Default</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>driverCombo</tabstop>
  <tabstop>databasePicker</tabstop>
  <tabstop>usernameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>hostnameEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SqlConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SqlConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
